lib/models.js

```diff
@@ -144,6 +144,8 @@ class Concept {
     this._system = system;
     this._code = code;
     this._display = display;
+    this._descendingFrom = null;
+    this._includesFrom = null;
```

Comment:
You could create a `ValueSetConceptRule` that `extends Concept` and adds those fields.  I think that would be better.  If you go, make sure you add getters for them too (I noticed them missing below).